### ISSUE TITLE
After a five successful execution I get the input as output

### ISSUE THREAD

**Issue by eickeBuecking** [microsoft/semantic-kernel#4116]
I am iterating over a set of files which are delivering the input for a semantic function. This works fine for the first five iterations, but from this point on I get as result of the semantic function the provided input. 

This is my code: 

```import os, time
import semantic_kernel as sk
from semantic_kernel.connectors.ai.open_ai import AzureChatCompletion

from Plugins.OrchestratorPlugin.OrchestratorPlugin import Orchestrator

async def main(folder_name : str):
    kernel = sk.Kernel()    

    deployment, api_key, endpoint = sk.azure_openai_settings_from_dot_env()
    kernel.add_chat_service("chat_completion", AzureChatCompletion(deployment, endpoint, api_key))

    plugins_directory = "./Plugins"

    invoice_extractor_plugin = kernel.import_semantic_skill_from_directory(
        plugins_directory,
        "InvoiceExtractorPlugin",
    )

    # iterate over txt files in folder and extract fields and store the result in the same folder with the same name but fields_ as prefix

    for file in os.listdir(folder_name):
        if file.endswith(".txt"):
            with open(folder_name + "/" + file, "r", encoding="utf-8") as f:
                text = f.read()
            variables = sk.ContextVariables()
            variables["input"] = text
            print(f"Start: {file}")
            result =  (await kernel.run_async(
                invoice_extractor_plugin["ExtractFields"],
                input_vars=variables,
            )).result.strip()

            with open(folder_name + "/fields_" + file, "w", encoding="utf-8") as f:
                f.write(result)

            print(f"Done: {file}")
   

if __name__ == "__main__":
    import asyncio
    asyncio.run(main(folder_name="C:/Users/user/development/temp"))```

**Comment by eickeBuecking** [comment]
Nevermind, it is a quota issue plus suboptimal error handling in Azure.

**Comment by hlums** [comment]
@eickeBuecking Can you explain how you fixed this? I'm getting the same issue the output is the same as input. The code "finishes" in a second. I suspect GPT was not invoked successfully.

**Comment by eickeBuecking** [comment]
In my case the API was invoked and replied with a error message with regards to the exceeded quota. But I only found out doing direct curls against the API, because the error message got lost somewhere. I assume the OpenAI-SDK is the culprit, because I saw the same behavior - returning the input document instead of an error - when directy using openai sdk, without semantic-kernel as a middleware in between. 
I solved by providing a higher contingent to the respective model.
